### ISSUE TITLE
Throwing boots at AI APIs

### ISSUE THREAD

**Issue by bohara2000** [NaPoGenMo/NaPoGenMo2024#2]
For now, I'm going to aim to play with throwing different data at one or more LLMs, then creating functions inspired by the Oulipo movement make something that is at least interesting in planning if not execution :)

**Comment by bohara2000** [comment]
I had this line of thought of emulating a digestive process: 

- Maceration (chewing up data/text - ingestion?)
- Digestion (breaking down/transformation)
- Extraction (refining)
- Excretion (discarding waste)

Also put down some ideas for ways to take the text apart.

![20240404_092034](https://github.com/NaPoGenMo/NaPoGenMo2024/assets/1751978/7c57a1dc-9db7-4ab9-a025-7fc7d07888eb)
![20240404_092230](https://github.com/NaPoGenMo/NaPoGenMo2024/assets/1751978/f9ec0dfd-99b3-416c-a1db-75fa81263e23)

**Comment by bohara2000** [comment]
These past few days have been spent designing, coding, then rethinking the design. I've got many of the main pieces together, but need to build (I think) two more.

**Comment by bohara2000** [comment]
So...very late, but I got something out there. I made heavy use of Microsoft Azure components like Logic Apps, Azure Functions, and Storage Accounts. Here are some examples of what it can generate:

https://github.com/NaPoGenMo/NaPoGenMo2024/assets/1751978/b11ca057-12c2-4380-9fb9-1ff6a54fede8

https://github.com/NaPoGenMo/NaPoGenMo2024/assets/1751978/722e6874-8683-47ed-8fa1-e00a544f4313

In the end, I put together something that generates - or attempts to generate - a poem three times a day, based on the weather from a random US city and state. The github repo only contains the python code. 

Hope someone finds this interesting/useful:
https://github.com/bohara2000/NaPoGenMo2024-PoeticDigestiveSystem
